src/Workspaces/Core/Portable/Classification/ClassifiedSpan.cs

```diff
@@ -1,5 +1,7 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+#nullable enable
```

Comment:
Nothing is blocking ClassifiedSpan being created with a null classification type -- is that something somebody could be doing? (I'm not sure if this is something third parties create or if they just consume this...)

Reply:
I believe that would use a "text" classification type.